ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -616,6 +616,18 @@ tests:
       COMPUTE_NODE_TYPE: t2a-standard-4
       OCP_ARCH: arm64
     workflow: openshift-e2e-gcp-ovn
+- as: ocp-e2e-upgrade-gcp-ovn-arm64
+  interval: 72h
```

Comment:
This is how often Prow will launch these jobs.  Unless someone will be watching these periodics directly, you may not want to run them this often.  You can set it higher (e.g. `interval: 168h` for "weekly" or `cron: '@yearly'` for "yearly") to avoid running the periodics as often, if you'll only be watching the release controllers.  Release controller runs will be launched when that stream gets a new release, regardless of what you set here.

Reply:
makes sense. let me review these numbers with the team (for all jobs) and see if we can come up with a good interval time.